domains/jiyath.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "Jiyath5516F",
+           "email": "",
+           "discord": "831836517516312586",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.Ou0aZJPtQR6dKH2nzSJA1hCmd52TrzxzUZWRL1JTg9EYUmyYBhVklrkaDqGV_aV8gKg64q7RbQ8Aa9Ghd5zI72IMp_k8noQQzr3i5DNv1ij001AtGFuUS7C_7nELc4hBrmJdVt02d8iH2CAM3Qg6jW1dNrSLRF6SMlU6Nx91ruWIEUayzHz30kG6mS9l-0SxJwMpDvmO8tsFqumPy_wqDQXePnqNwVGQK3UeZscSXYTXEQrVaJKgrjKzwJovgnrussMRO6Mg6bVQcMp80JFts9mXtr5DQ7ZBG2o-WY4y2X7ee1aFmAvJgtw8z24asocNYiYVKy4d7rzuTS9AIIJe-Q.qoTJuB_rN7r61yHm_SQ1sQ.pwH7RHNzGpV365BBO_tolwajA2XArfIV0gdLJR0y40VRgabi70frRH49rx62T_n3voJu11dQeX4-PusUA21JK3ATebpwo557CA9Za9LY3hw.JGAHTASzBs0rdG7I7qLs7Q"
+        },
+    
+        "record": {
+            "A": ["185.199.108.153"]
```

Comment:
I can not reach your website, 404, if you use github pages, you need to put all 4 IPs

Reply:
https://jiyath5516f.github.io/jiyath.is-a.dev/